src/components/progress.js

```diff
@@ -162,27 +165,21 @@ class MiniMediaPlayerProgress extends LitElement {
       .mmp-progress__duration__remaining {
         opacity: .5;
       }
-      paper-progress {
+      .progress-bar {
         height: var(--mmp-progress-height);
-        --paper-progress-height: var(--mmp-progress-height);
         bottom: 0;
         position: absolute;
         width: 100%;
```

Comment:
```suggestion
        width: 0;
```

This needs to be changed to 0 so that the bar is not filled with `media_duration: 0`:

<img src="https://github.com/kalkih/mini-media-player/assets/11841379/56aeaef6-9f2b-4ec4-8a4e-8424194676d3">

Media player state:
```yaml
media_duration: 0
media_position: 114.841
media_position_updated_at: "2023-12-03T13:17:20.041398+00:00"
```